.travis.yml

```diff
@@ -6,7 +6,7 @@ git:
   depth: 10
 
 rvm:
-  - 2.3.8
   - 2.4.6
   - 2.5.5
   - 2.6.3
+  - 2.7.1
```

Comment:
For future debugging purposes, can we put CI changes in a separate PR?

Reply:
sure, let me bring it in separate PR then

Reply:
https://github.com/github/rubocop-github/pull/58

Reply:
It looks like we'll need to do this PR first, then the CI PR since CI is breaking on the warnings :)

Reply:
yes, but I got CI test failure on 2.3 for this PR :(

Reply:
It looks like on 2.3 it failed with `Layout/IndentationStyle`:

https://travis-ci.org/github/github/rubocop-github/jobs/676500683

Which is odd, since you fixed that. I didn't chase it further (maybe there's another place where it's being used?)